src/jquery.dataTables.yadcf.js

```diff
@@ -1884,67 +1884,139 @@ if (!Object.entries) {
+					})).append($("<input>", {
+						type: "checkbox",
+						title: columnObj.exclude_label,
+						onclick: "yadcf.stopPropagation(event);yadcf.rangeNumberKeyUP('" + table_selector_jq_friendly + "',event);"
+					}));
 				} else {
-					exclude_str = '<span class="yadcf-exclude-wrapper" onmousedown="yadcf.stopPropagation(event);" onclick="yadcf.stopPropagation(event);">' +
-						'<div class="yadcf-label small">' + columnObj.exclude_label + '</div><input type="checkbox" title="' + columnObj.exclude_label + '" onclick="yadcf.stopPropagation(event);"></span>';
-				}
-			}
-
-			null_str = '';
+					exclude_str = $("<span>", {
+						class: "yadcf-exclude-wrapper",
+						onmousedown: "yadcf.stopPropagation(event);",
+						onclick: "yadcf.stopPropagation(event);"
+					}).append($("<div>", {
+						class: "yadcf-label small",
+						text: columnObj.exclude_label
+					})).append($("<input>", {
+						type: "checkbox",
+						title: columnObj.exclude_label,
+						onclick: "yadcf.stopPropagation(event);"
+					}));
+				}
+			}
+
+			null_str = $();
 			if (columnObj.null_check_box === true) {
 				if (columnObj.externally_triggered !== true) {
-					null_str = '<span class="yadcf-null-wrapper" onmousedown="yadcf.stopPropagation(event);" onclick="yadcf.stopPropagation(event);">' +
-						'<div class="yadcf-label small">' + columnObj.null_label + '</div><input type="checkbox" title="' + columnObj.null_label + '" onclick="yadcf.stopPropagation(event);yadcf.nullChecked(event,\'' + table_selector_jq_friendly + '\',' + column_number + ');"></span>';
+					null_str = $("<span>", {
+						class: "yadcf-null-wrapper",
+						onmousedown: "yadcf.stopPropagation(event);",
+						onclick: "yadcf.stopPropagation(event);"
+					}).append($("<div>", {
+						class: "yadcf-label small",
+						text: columnObj.null_label
+					})).append($("<input>", {
+						type: "checkbox",
+						title: columnObj.null_label,
+						onclick: "yadcf.stopPropagation(event);yadcf.nullChecked(event,'" + table_selector_jq_friendly + "'," + column_number + ");"
+					}));
 				} else {
-					null_str = '<span class="yadcf-null-wrapper" onmousedown="yadcf.stopPropagation(event);" onclick="yadcf.stopPropagation(event);">' +
-						'<div class="yadcf-label small">' + columnObj.null_label + '</div><input type="checkbox" title="' + columnObj.null_label + '" onclick="yadcf.stopPropagation(event);yadcf.nullChecked(event,\'' + table_selector_jq_friendly + '\',' + column_number + ');"></span>';
+					null_str = $("<span>", {
+						class: "yadcf-null-wrapper",
+						onmousedown: "yadcf.stopPropagation(event);",
+						onclick: "yadcf.stopPropagation(event);"
+					}).append($("<div>", {
+						class: "yadcf-label small",
+						text: columnObj.null_label
+					})).append($("<input>", {
+						type: "checkbox",
+						title: columnObj.null_label,
+						onclick: "yadcf.stopPropagation(event);yadcf.nullChecked(event,'" + table_selector_jq_friendly + "'," + column_number + ");"
+					}));
```

Comment:
Both variants of `null_str` are the same, looks like a bug. I kept the original code since I'm not sure what would be the correct version.

Reply:
it was a PR, indeed looks the same assignment, feel free to modify / remove duplicates